types/lodash/scripts/generate-modules.ts

```diff
@@ -19,7 +19,7 @@ async function main(): Promise<void> {
     }
 
     // Generate lodash/tsconfig.json
-    fs.writeFileSync(path.join("..", "tsconfig.json"), lodashTsconfig(all));
+    fs.writeFileSync(path.join("..", "tsconfig.json"), lodashTsconfig());
```

Comment:
Maybe we should stop auto-generating this file? If its contents are not based on the list of modules, then I'm inclined to let people edit it manually instead of auto-generating it.

Reply:
I agree, but am not familiar with the history and why it's generated by the script, then not sure if it's OK...